core/src/main/java/ai/timefold/solver/core/preview/api/domain/variable/declarative/ShadowVariablesInconsistent.java

```diff
@@ -30,18 +30,18 @@
  * </ul>
  * <p>
  * Should be used in a filter for a hard {@link Constraint} to penalize
- * looped entities, since {@link PlanningSolution} with looped entities are
+ * inconsistent entities, since {@link PlanningSolution} with inconsistent entities are
  * typically not valid.
  * <p>
  * Important:
- * Do not use a {@link ShadowVariableLooped} property in a method annotated
- * with {@link ShadowSources}. {@link ShadowVariableLooped} properties can
+ * Do not use a {@link ShadowVariablesInconsistent} property in a method annotated
+ * with {@link ShadowSources}. {@link ShadowVariablesInconsistent} properties can
  * be updated after the {@link ShadowSources} marked method is called, causing
  * score corruption. {@link ShadowSources} marked methods do not need to check
- * {@link ShadowVariableLooped} properties, since they are only called if all
+ * {@link ShadowVariablesInconsistent} properties, since they are only called if all
  * their dependencies are not looped.
```

Comment:
I suggest doing a check on the word "loop", as clearly some of them slipped through the cracks.

Also, instead of saying "if all of their dependencies are not looped", the human way of speaking would be "if all of their dependencies are consistent."
"Inconsistent" has a good inverse, whereas "looped" did not - we can use that inverse.